### PR TITLE
Update localization.ru.lua

### PR DESCRIPTION
**Pull request by Hollicsh**
Add phrases. Translation later.

### CODE REVIEW

**Review by MysticalOS — CHANGES_REQUESTED** [review]
no reason to add placeholders, i'll leave this open for now but overwriting english ones with english ones isn't needed. You'll want the actual RU text or better to leave blank or commented out for now.

**Comment by Molkree** [comment]
Wowhead has these phrases.
[Xy'noc](https://ru.wowhead.com/npc=176758)
Докучают разъяренные звери? У нас есть решение!
[Venza Goldfuse](https://ru.wowhead.com/npc=176705)
Вот мой шанс! Топор будет моим!
[Al'dalil](https://ru.wowhead.com/npc=178074)
Прости нас за вторжение, Со'лея. Кажется, сейчас не самое подходящее время.

**Comment by Hollicsh** [comment]
Yeah. Done.

File: DBM-Party-Shadowlands/localization.ru.lua [hunk 1]
```diff
@@ -306,7 +306,12 @@ L:SetGeneralLocalization({
 -----------------------
 -- The Menagerie --
 -----------------------
---L= DBM:GetModLocalization(2454)
+L= DBM:GetModLocalization(2454)
+
+L:SetMiscLocalization({
+ 	AchilliteRPTrigger	= "Докучают разъяренные звери? У нас есть решение!",
+ 	VenzaRPTrigger		= "Вот мой шанс! Топор будет моим!"
```
Comment (Molkree) [inline DBM-Party-Shadowlands/localization.ru.lua]:
```suggestion
	AchilliteRPTrigger	= "Докучают разъяренные звери? У нас есть решение!",
	VenzaRPTrigger		= "Вот мой шанс! Топор будет моим!"
```

File: DBM-Party-Shadowlands/localization.ru.lua [hunk 2]
```diff
@@ -321,7 +326,11 @@ L:SetGeneralLocalization({
 -----------------------
 -- So'azmi --
 -----------------------
---L= DBM:GetModLocalization(2451)
+ L= DBM:GetModLocalization(2451)
+
+ L:SetMiscLocalization({
+ 	RPTrigger	= "Прости нас за вторжение, Со'лея. Кажется, сейчас не самое подходящее время."
```
Comment (Molkree) [inline DBM-Party-Shadowlands/localization.ru.lua]:
```suggestion
	RPTrigger	= "Прости нас за вторжение, Со'лея. Кажется, сейчас не самое подходящее время."
```

**Review by Molkree — CHANGES_REQUESTED** [review]
You've added SPACEs before TABs accidentally.

**Comment by MysticalOS** [comment]
I also want to add that wowhead quotes are not same as in game verified ones either. Sometimes those are typed up by users and not copy/pasted so they can have missing spaces or punctuation. Since these are actual trigger strings they need to be verified from game or they might note even work.

**Comment by Hollicsh** [comment]
Let's leave it like this for now.

**Comment by Hollicsh** [comment]
I checked all the phrases in the game, they match 100%.

**Review by MysticalOS — APPROVED** [review]

**Pull request merged**